ticdc/integrate-confluent-using-ticdc.md

```diff
@@ -1,130 +1,264 @@
     ```
 
-3. Deploy TiCDC in one of the following ways. If TiCDC is already deployed, you can skip this step.
+    You can refer to [Manage TiCDC Cluster and Replication Tasks](/ticdc/manage-ticdc.md) to manage the changefeed.
 
-    - [Deploy a new TiDB cluster that includes TiCDC using TiUP](/ticdc/deploy-ticdc.md#deploy-a-new-tidb-cluster-that-includes-ticdc-using-tiup)
-    - [Add TiCDC to an existing TiDB cluster using TiUP](/ticdc/deploy-ticdc.md#add-ticdc-to-an-existing-tidb-cluster-using-tiup)
-    - [Add TiCDC to an existing TiDB cluster using binary (not recommended)](/ticdc/deploy-ticdc.md#add-ticdc-to-an-existing-tidb-cluster-using-binary-not-recommended)
+### Step 4. Write data to generate change logs
 
-    Make sure that your TiDB and TiCDC clusters are healthy before proceeding.
+After the preceding steps are done, TiCDC sends change logs of incremental data in the TiDB cluster to Confluent Cloud. This section describes how write data to generate change logs.
 
-4. Create a `changefeed` by running the `cdc cli` command:
+1. Simulate service workload.
 
-    {{< copyable "shell-regular" >}}
+    In the lab environment, to generate change logs, you can use go-tpc to write data to the TiDB cluster. Specifically, run the following command to create a database `tpcc` in the TiDB cluster. Then use TiUP bench to write data to this new database.
 
     ```shell
-    ./cdc cli changefeed create --pd="http://127.0.0.1:2379" --sink-uri="kafka://127.0.0.1:9092/testdb_test?protocol=avro" --opts "registry=http://127.0.0.1:8081"
+    tiup bench tpcc -H 127.0.0.1 -P 4000 -D tpcc --warehouses 4 prepare
+    tiup bench tpcc -H 127.0.0.1 -P 4000 -D tpcc --warehouses 4 run --time 300s
     ```
 
-    > **Note:**
-    >
-    > Make sure that PD, Kafka, and Schema Registry are running on their respective default ports.
+    For more details about go-tpc, refer to [How to Run TPC-C Test on TiDB](/benchmark/benchmark-tidb-using-tpcc.md).
+
+2. Observe data in Confluent Cloud.
+
+    ![Confluent topics](/media/integrate/confluent-topics.PNG)
 
-## Test data replication
+    In the Confluent Cloud Console, click **Topics**. You can see that the target topics have been created and are receiving data. At this time, incremental data of the TiDB database is successfully replicated to Confluent Cloud.
 
-After TiDB is integrated with Confluent Platform, you can follow the example procedures below to test the data replication.
+## Integrate data with ksqlDB
 
-1. Create the `testdb` database in your TiDB cluster:
+ksqlDB is a database purpose-built for stream processing applications. You can create ksqlDB clusters on Confluent Cloud and access incremental data replicated by TiCDC.
 
-    {{< copyable "sql" >}}
+1. Select **ksqlDB** in the Confluent Cloud Console and create a ksqlDB cluster as instructed.
+
+    Wait until the ksqlDB cluster status is **Running**. This will takes several minutes.
```

Comment:
```suggestion
    Wait until the ksqlDB cluster status is **Running**. This will take several minutes.
```

Reply:
Fixed in [3fa1b2c](https://github.com/pingcap/docs/pull/9866/commits/3fa1b2c665c117963cd01ccdc9574a30475dada0)